### PR TITLE
Adaugare fisiere css pentru 2 dintre fisierele html

### PR DESCRIPTION
**Pull request by andreeaNeamtiu**

### CODE REVIEW

File: newEvaluation.css [hunk 1]
```diff
@@ -13,5 +13,91 @@ li
 
 fieldset
 {
-	max-width: 850px;
+	/*max-width: 850px;*/
```
Comment (softvision-alexcarcea) [inline newEvaluation.css:16]: If this is no longer needed, might as well go ahead and remove it altogether.

File: newEvaluation.css [hunk 2]
```diff
@@ -13,5 +13,91 @@ li
...
+}
+
+section
+{
+	width: 90%;
+}
+
+form
+{
+	flex-direction: column-reverse;
```
Comment (softvision-alexcarcea) [inline newEvaluation.css:27]: Could you explain what this rule does?

File: newEvaluation.css [hunk 3]
```diff
@@ -13,5 +13,91 @@ li
...
+
+legend, h5
+{
+	color: #03A9F4;
+}
+
+textarea
+{
+	resize: none;
+	height: 92.6%;
```
Comment (softvision-alexcarcea) [inline newEvaluation.css:66]: Hmm, interesting percentage! How did you come up with this particular value?

File: newEvaluation.html [hunk 4]
```diff
@@ -14,6 +14,7 @@
 	</nav>
 	
 	<!-- Section1 -->
+	<div id = "fieldsetContainer">
```
Comment (softvision-alexcarcea) [inline newEvaluation.html:17]: This is more a matter of personal preference than anything else, as HTML is pretty permissive, but I personally feel that the extra spaces around the `=` makes the markup harder to read. Call me old-fashioned. :)

Comment (softvision-alexcarcea) [reply newEvaluation.html:17]: If you feel that it works better for you this way, by all means, carry on!

File: newEvaluation.html [hunk 5]
```diff
@@ -57,7 +58,7 @@
 	<!-- Section3 -->
 	<section>
 		<h5>Should the candidate be hired?</h5>
-		<textarea rows = "5" cols = "" placeholder = "The type of project that is suitable for the candidate. Is guidance required for the candidate"></textarea>
+		<textarea rows = "5" cols = "" placeholder = "The type of project that is suitable for the candidate.&#10;Is guidance required for the candidate"></textarea>
```
Comment (softvision-alexcarcea) [inline newEvaluation.html:61]: What is `&#10;`?

File: newEvaluation.html [hunk 6]
```diff
@@ -69,7 +70,7 @@
 	<!-- Section5 -->
 	<section>
 		<h5>Workflow, Leadership & Soft Skills</h5>
```
Comment (softvision-alexcarcea) [inline newEvaluation.html:72]: Weird... why might this `&` character be showing up on red background in GitHub? :P

File: newEvaluation.html [hunk 7]
```diff
@@ -379,10 +380,14 @@
 					</li>
 				</ul>
 			</fieldset>
-			<button type = "submit" name = "submit">Submit</button>
+			<button id = "submitButton" type = "submit" name = "submit">SUBMIT</button>
```
Comment (softvision-alexcarcea) [inline newEvaluation.html:383]: What if I told you that you could affect the casing of a piece of text through CSS alone? What if I also told you that you could adjust the spacing around elements with CSS, as well, and that tags like `<br>` are a bad practice, since they force information at the data layer to be presented a certain way across all environments / devices / screen sizes etc? What if I also told you that for mobile devices with less screen real estate, the presentation often differs slightly, so as to be able to fit more information in less space?

File: style.css [hunk 8]
```diff
@@ -3,28 +3,41 @@ body
...
   background: #FFFFFF;
-  position: relative;
+  font-family: 'Roboto', sans-serif;
+  font-size: 13px;
+  width: 100%;
+}
+
+.logoPicture
+{
+	width: 9%;
```
Comment (softvision-alexcarcea) [inline style.css:13]: Why `9%`, particularly?

File: style.css [hunk 9]
```diff
@@ -3,28 +3,41 @@ body
...
 }
 
 nav a
 {
-	color: #686868;
+	color: #636262;
+	width: 10%;
+	padding-right: 40px;
+	padding-left: 40px;
+	/*float: left;*/
```
Comment (softvision-alexcarcea) [inline style.css:29]: If not needed, might want to go ahead and remove.

File: style.css [hunk 10]
```diff
@@ -3,28 +3,41 @@ body
...
 	border-bottom: 1px solid #d9dadb;
+	justify-content: space-around;
 }
 
 nav a
 {
-	color: #686868;
+	color: #636262;
+	width: 10%;
+	padding-right: 40px;
```
Comment (softvision-alexcarcea) [inline style.css:27]: Would it be possible to bundle these two padding values in a single line? If so, how?

File: style.css [hunk 11]
```diff
@@ -3,28 +3,41 @@ body
...
+	padding-left: 40px;
+	/*float: left;*/
+	white-space: nowrap;
```
Comment (softvision-alexcarcea) [inline style.css:30]: What does this rule do and why do you need it?

File: style.css [hunk 12]
```diff
@@ -33,30 +46,55 @@ a
...
 
-.footerDiv {
-	position: fixed;
-	right: 0;
-	bottom: 0;
-	left: 0;
-	padding-right: 20;
-	background-color: #efefef;
-	text-align: right;
+button
```
Comment (softvision-alexcarcea) [inline style.css:49]: Any particular reason why we're styling against raw tags here? Not saying there's anything wrong with it, just asking for a bit of context. :)

**Review by softvision-alexcarcea — APPROVED** [review]

File: newEvaluation.css (hunk 1 above)
Comment (andreeaNeamtiu) [reply newEvaluation.css:16]: Ok, I will.

File: newEvaluation.css (hunk 2 above)
Comment (andreeaNeamtiu) [reply newEvaluation.css:27]: I was thinking that it will align vertically the sections inside the form. I also tried on a div container but apparently i'm not sure yet how to use it.

File: newEvaluation.css (hunk 3 above)
Comment (andreeaNeamtiu) [reply newEvaluation.css:66]: I thought that if i use percentage it will be a little bit responsive and I modified the percentage till it was at the same size as the fieldset below.

File: newEvaluation.html (hunk 4 above)
Comment (andreeaNeamtiu) [reply newEvaluation.html:17]: I will give it a try.  It should be readable for most of the people not just for me and i'm sure that if from your experience this way is unusual it means that there is not too often used. :)

File: newEvaluation.html (hunk 5 above)
Comment (andreeaNeamtiu) [reply newEvaluation.html:61]: i used it to break row inside placeholder. I first tried br and /n but this were not working good because it were shown on the page.

File: newEvaluation.css (hunk 3 above)
Comment (softvision-alexcarcea) [reply newEvaluation.css:66]: Question is: does it stay the same size if you resize the window? :)

Comment (andreeaNeamtiu) [reply newEvaluation.css:66]: No no, it doesn't :) I will try with px instead of percentage for width and I found that in this case I don't need height because i already have the rows property in html.

File: newEvaluation.html (hunk 6 above)
Comment (andreeaNeamtiu) [reply newEvaluation.html:72]: Should I use something else instead? Is it misinterpreted by the browser?

File: newEvaluation.html (hunk 7 above)
Comment (andreeaNeamtiu) [reply newEvaluation.html:383]: Is it better to add margin-bottom instead of br tags?

File: style.css (hunk 10 above)
Comment (andreeaNeamtiu) [reply style.css:27]: maybe like this? padding: 0px 40px 0px 40px;

File: style.css (hunk 9 above)
Comment (andreeaNeamtiu) [reply style.css:29]: ok

File: style.css (hunk 11 above)
Comment (andreeaNeamtiu) [reply style.css:30]: It prevents text from wrapping to another line. I was styling the menu bar when I saw that the "new evaluation" button wrap when resizing the window so i thought that i should stop it.

File: style.css (hunk 10 above)
Comment (zuldy) [reply style.css:27]: Can you think of a more shorter version?

Comment (andreeaNeamtiu) [reply style.css:27]: padding: 0px 40px;  :)

File: newEvaluation.html (hunk 6 above)
Comment (softvision-alexcarcea) [reply newEvaluation.html:72]: Modern browsers have pretty good mechanisms to try to infer what you mean, even if the actual markup isn't 100% valid. In this case, the issue stems from the fact that in HTML, `&` is a reserved character, as it denotes the start of an HTML entity. In order to include a literal "&" in your page, you'd need to encode it as `&amp;` - the HTML entity denoting the ampersand character, "&".

File: newEvaluation.html (hunk 7 above)
Comment (softvision-alexcarcea) [reply newEvaluation.html:383]: Most definitely! Imagine this: you need to develop a site that works well when viewed on desktops, tablets and mobile devices. For desktops, you have enough screen real estate and can include enough spacing in your page, no probz. So you put a bunch of `<br>`s in there and it looks fantastic!

For tablets, less space available, your `<br>`s still force the layout breaks, a bit too much spacing for your liking, but hey, it's not _that bad_, is it? But then, you try to view the page on your brand new, shiny smartphone... Some of this stuff's full of content, then suddenly you're scrolling through half a page worth of empty space... _"What's happening here?"_, you ask... well, your `<br>`s are still stack in that markup, still forcing the layout engine to break to a new line, and even media queries don't help you at this point, without some ugly hacks in there to make everything play along...

Long story short? Yep, **always** pick `margin-bottom` instead of `<br>` tags, _unless_ you're absolutely certain that a `<br>` tag is what you need (in which case, you might _still_ want to check with one of the trainers to make sure your assumption is correct :P).

File: style.css (hunk 10 above)
Comment (softvision-alexcarcea) [reply style.css:27]: That's what I call team effort! :D
